gen-ai/sample-claude-multimodal.json

```diff
@@ -0,0 +1,108 @@
+{
+  "variable": [
+    {
+      "value": "https://api.anthropic.com/v1/messages",
+      "key": "api.url"
+    },
+    {
+      "value": "claude-3-opus-20240229",
```

Comment:
I really wish there was a way to include comments here with haiku and sonnet...

Reply:
What if we added a console log statement? Would that be too subtle??